bin/node/cli/src/service.rs

```diff
@@ -47,7 +48,7 @@ pub fn new_partial(
 ) -> Result<sc_service::PartialComponents<
 	FullClient, FullBackend, FullSelectChain,
 	sp_consensus::DefaultImportQueue<Block, FullClient>,
-	sc_transaction_pool::FullPool<Block, FullClient>,
+	sc_transaction_pool::FullPool<Block, FullClient, SpawnTaskHandle>,
```

Comment:
I'm _not_ a fan of this generic getting propagated. It might be best to use a `Box<dyn SpawnNamed>` in the tx pool.

Reply:
Alternatively, we could move `SpawnTaskHandle` into it's own `sp` crate.

Reply:
`Box<dyn SpawnNamed>` is fine IMO